proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+
+**Acceptance Criteria**
+To be considered for Sandbox Stage, the project must meet the following requirements:
+* 2 CPC sponsors to champion the project & provide mentorship
+* A presentation to at the meeting of the CPC
+* Adherence to the _$FOUNDATION_ IP Policy
+* Upon acceptance, Sandbox projects must list their status prominently on website/readme
+
+### Incubating Stage*
+**Definition** 
+The Incubation Stage is for projects that are interested in reaching the Top-Level Stage, or want to have more access, participation, responsibilities and representation in the CPC. Incubating projects will receive mentorship from the CPC and are expected to actively develop their community of contributors, governance, and other project documentation. 
+
+**Examples**
+1. Projects that are on their way or very likely to become Top Level Projects.
+
+**Expectations**
+Projects can remain in the Incubating stage indefinitely, but they are generally expected to graduate within two years.
+
+**Acceptance Criteria**
+To be considered for Incubation Stage, the project must meet the Sandbox requirements as well as the following:
+ * Document that it is being used successfully in production by at least three independent end users which, in the CPC’s judgement, are of adequate quality and scope.
+ * Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+ * Demonstrate a substantial ongoing flow of commits and merged contributions.
+ * Since these metrics can vary significantly depending on the type, scope and size of a project, the CPC has final judgement over the level of activity that is adequate to meet these criteria.
+ * Recieve a two-thirds supermajority vote of the CPC to move to Incubation Stage. 
+
+
+### Top Level/Graduate Stage*
+**Definition**
+
+**Examples**
+
+**Expectations**
+
+**Acceptance Criteria**
+To graduate from Sandbox or Incubating status, or for a new project to join as a Top-Level project, a project must meet the Incubating stage criteria plus:
+
+ * Have committers from at least two organizations.
+ * Adopt the _$FOUNDATION_ Code of Conduct.
+ * Explicitly define a project governance and committer process. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
+ * Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website).
+ * Receive a supermajority vote from the CPC to move to Graduation stage. Projects can attempt to move directly from Sandbox to Top Level, if they can demonstrate sufficient maturity. 
+
+
+### Archive/Emeritus Stage*
+**Definition**
+
+**Examples**
+
+**Expectations**
+
+**Acceptance Criteria**
+* Archiving criteria: projects may be archived via a 2/3 vote from the CPC
```

Comment:
...and agreement from project ownership?

Reply:
that makes sense to me

Reply:
We'd need to make sure we cover the case where it is being archived due to "lack of project ownership" as well.

Reply:
I agree - we could ask the CPC to develop some protocol for the case that a project maintainer has stepped down & no one has taken their place